Containers/nextcloud/config/s3.config.php

```diff
@@ -16,12 +16,12 @@
         'port' => getenv('OBJECTSTORE_S3_PORT') ?: '',
         'storageClass' => getenv('OBJECTSTORE_S3_STORAGE_CLASS') ?: '',
         'objectPrefix' => getenv("OBJECTSTORE_S3_OBJECT_PREFIX") ? getenv("OBJECTSTORE_S3_OBJECT_PREFIX") : "urn:oid:",
-        'autocreate' => (strtolower($autocreate) === 'false' || $autocreate == false) ? false : true,
-        'use_ssl' => (strtolower($use_ssl) === 'false' || $use_ssl == false) ? false : true,
+        'autocreate' => (strtolower($autocreate) === 'false' || $autocreate === false) ? false : true,
+        'use_ssl' => (strtolower($use_ssl) === 'false' || $use_ssl === false) ? false : true,
         // required for some non Amazon S3 implementations
-        'use_path_style' => $use_path == true && strtolower($use_path) !== 'false',
+        'use_path_style' => $use_path === true && strtolower($use_path) !== 'false',
         // required for older protocol versions
-        'legacy_auth' => $use_legacyauth == true && strtolower($use_legacyauth) !== 'false'
+        'legacy_auth' => $use_legacyauth === true && strtolower($use_legacyauth) !== 'false'

```

Comment:
This won't work. Neither variable will be ever be true so this condition will always be false.

Either revert those changes or use the following:

```php
'use_path_style' => strtolower($use_path) === 'true',
'legacy_auth' => strtolower($use_legacyauth) === 'true'
```

Here, the default value is false so we need to check if the variable is set to `"true"`.